spec/requests/admin/configs_spec.rb

```diff
@@ -48,13 +48,12 @@
       end
 
       it "updates site config admin action taken" do
-        Timecop.freeze do
-          expect(SiteConfig.admin_action_taken_at).not_to eq(5.minutes.ago)
-          allow(SiteConfig).to receive(:admin_action_taken_at).and_return(5.minutes.ago)
-          post "/admin/config", params: { site_config: { health_check_token: "token" },
-                                          confirmation: confirmation_message }
-          expect(SiteConfig.admin_action_taken_at).to eq(5.minutes.ago)
-        end
+        expect do
+          post "/admin/config", params: {
```

Comment:
This example isn't really related to the PR's main change but it was giving me trouble and the use of `Timecop` here seemed superfluous relative to a regular `change` assertion.